### PR TITLE
[next] Update SDK next dependency

### PR DESCRIPTION
**Pull request by crawr**
Signed-off-by: Richelle Anne Craw <richelleanne.craw@broadcom.com>

## Release Notes

Milestone:

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [ ] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Reminder

After a PR is merged into the `master` branch, create a PR from `master` to the `next` branch resolving any conflicts.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by JillieBeanSim** [comment]
This PR looks good to me with the version updates and I don't see any issues in VSC.  Not sure why the theia tests are failing only on this branch and passing on others though.  We are still pulling in Theia 1.20.0 released a month ago.

**Review by jellypuno — DISMISSED** [review]

File: package.json
Comment (zFernand0) [inline package.json:21]: Thanks for updating.

Since the CLI is being more careful when making schema changes to a vNext version, do we want to consider automating this kind of PRs ?
Just a thought 😋

**Review by zFernand0 — DISMISSED** [review]

**Comment by JillieBeanSim** [comment]
I know this PR has the approvals needed to merge, but do we still want to investigate the Theia test breakages before merging it in? @jellypuno @crawr @zFernand0

**Comment by crawr** [comment]
Theia tests have been fixed but there seems to be some new audit issues found.

The theia breakage was due to the new cert file and cert key file input field prompts during profile creation.

**Comment by zFernand0** [comment]
> Theia tests have been fixed but there seems to be some new audit issues found.
> 
> The theia breakage was due to the new cert file and cert key file input field prompts during profile creation.

Thank you! 😋 

It seems that the markdown-it package is updated in the latest versions of the CLI (and imperative)

Maybe we can update them again (either on this PR or in a future one)

**Review by zFernand0 — DISMISSED** [review]

**Comment by JillieBeanSim** [comment]
@crawr if you could fix the conflicts in the am I can approve this and merge it in.  I am pulling in the latest cli next release in https://github.com/zowe/vscode-extension-for-zowe/pull/1637 for the login/logout with config files work.

**Review by JillieBeanSim — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**
